src/olympia/promoted/tests/test_commands.py

```diff
@@ -412,3 +412,38 @@ def disable_post_save_promoted_addon(self):
             sender=self.promoted_addon_class,
             dispatch_uid='addons.sync_promoted.promoted_addon_proxy',
         )
+
+
+class TestPromoteByFirefoxThemesCommand(TestCase):
+    def test_basic(self):
+        user = user_factory(pk=settings.TASK_USER_ID)
+        non_affected = [
+            # Not By Firefox user.
```

Comment:
Nit: more explicit would be instead of comment, set the user and the type like you are setting the status.